wire-ios/Wire-iOS/Sources/UserInterface/CustomAppLock/Unlock/UnlockViewController.swift

```diff
@@ -32,7 +32,7 @@ protocol UnlockViewControllerDelegate: AnyObject {
 /// This VC should be wrapped in KeyboardAvoidingViewController as the "unlock" button would be covered on 4 inch iPhone
 final class UnlockViewController: UIViewController {
 
```

Comment:
It'd be better if we only use `UserSession` and not the other protocols.

Reply:
I've done that in part 6. So for now we're good! @johnxnguyen